.gitignore

```diff
@@ -29,3 +29,4 @@ __pycache__
 .classpath
 .settings
 .jython_cache
+/.mypy_cache/
```

Comment:
What does the leading `/` mean?

Reply:
It means to ignore it only at the top-level (IDE did it for me... I guess that removing the leading `/` to ignore everywhere would be ok too).